### PR TITLE
Add: INEI from PERU

### PR DESCRIPTION
**Pull request by romellfudi**
- [X] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [X] My addition is ordered alphabetically
- [X] My submission has a useful description
- [X] The description does not end with punctuation
- [X] Each table column is padded with one space on either side
- [X] I have searched the repository for any relevant issues or pull requests
- [X] Any category I am creating has the minimum requirement of 3 items
- [X] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -634,6 +634,7 @@ API | Description | Auth | HTTPS | CORS |
 | [FEC](https://api.open.fec.gov/developers/) | Information on campaign donations in federal elections | `apiKey` | Yes | Unknown |
 | [Federal Register](https://www.federalregister.gov/reader-aids/developer-resources) | The Daily Journal of the United States Government | No | Yes | Unknown |
 | [Food Standards Agency](http://ratings.food.gov.uk/open-data/en-GB) | UK food hygiene rating data API | No | No | Unknown |
+| [INEI](http://iinei.inei.gob.pe/microdatos/) | Peruvian Statistical Government Open Data | No | Yes | Unknown |
```
Comment (matheusfelipeog) [inline README.md:637]: This link does not support HTTPS protocol.

```suggestion
| [INEI](http://iinei.inei.gob.pe/microdatos/) | Peruvian Statistical Government Open Data | No | No | Unknown |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your new contribution @romellfudi :)

Can you resolve this review, please?

**Comment by romellfudi** [comment]
I commit your correct suggestion

**Comment by pi-portharCommunity** [comment]
I cannot resolve it. Are you a developer?

**Comment by cameron187187** [comment]
this is my account some is stealing my funds and on my account!! please help. other account is xan187 if i lose my account

**Comment by romellfudi** [comment]
@matheusfelipeog It looks great, doesn't it?

**Review by matheusfelipeog — APPROVED** [review]
> Hello @matheusfelipeog It looks great, doesn't it?

@romellfudi That's perfect. A big THANK YOU for that. 😄

**Pull request merged**